endpointmanager/pkg/endpointmanager/fhirendpoint.go

```diff
@@ -40,3 +41,21 @@ func (e *FHIREndpoint) Equal(e2 *FHIREndpoint) bool {
 
 	return true
 }
+
+// Prepends url with https:// and appends with metadata/ if needed
```

Comment:
I don’t think we want this function to add metadata to the url, particularly because we’ll be querying the well known smart-on-fhir endpoint. I think the addition of metadata should be close to when the query happens.

Reply:
this function ins only called immediately before querying occurs. i can add a parameter to the function that will control whether or not the metadata check and addition occurs so that this can be reused for smart on fhir URLs too.